ckanext/unaids/tests/test_plugin.py

```diff
@@ -0,0 +1,49 @@
+"""Tests for plugin.py."""
+# encoding: utf-8
+
+import ckan.plugins
+from ckan.tests.helpers import call_action, reset_db
+from ckan.tests import factories
+import ckan.tests.helpers as helpers
+from ckanext.validation.model import create_tables, tables_exist
+import logging
+
+
+class TestPlugin(helpers.FunctionalTestBase):
+    '''Tests for the ckanext.example_iauthfunctions.plugin module.
+
+    Specifically tests that overriding parent auth functions will cause
+    child auth functions to use the overridden version.
+    '''
+    def setup(self):
```

Comment:
You don't need to use custom setup and teardown methods if using `helpers.FunctionalTestBase`. The parent class will clear and setup db for you. All you need to do is define which plugins you need with a field:
look at example in: ckanext-restricted/ckanext/restricted/tests/test_plugin.py
```
class TestRestrictedPlugin(helpers.FunctionalTestBase):

    _load_plugins = ['restricted']

    def test_restricted_resource_update(self):
(...)
```